data/il/legislature/Brian-W-Stewart-bdb2b767-7c04-4a7e-941b-9734155329d6.yml

```diff
@@ -11,18 +11,10 @@ roles:
   type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:il/government
   district: '89'
-- type: upper
+- end_date: '2023-01-10'
```

Comment:
@jessemortenson this person should actually be in the retired folder. They end their time in the senate on 01/10/2023 and are not serving elsewhere